packages/google_sign_in/google_sign_in/ios/Classes/google_sign_in-umbrella.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
The file was previously generated by CocoaPods with the same name.  Now we need to manage it manually and check it into the repo because we're defining a module map, which takes us out of the magical auto module world.